process_report/processors/prepayment_processor.py

```diff
@@ -194,6 +198,26 @@ def _apply_prepayments(self):
                         debit_entry_mask, invoice.PREPAY_DEBIT_FIELD
                     ] = prepay_amount_used
 
+    def _get_prepay_credits_snapshot(self):
+        managed_groups_list = list()
+        for group_name, group_dict in self.group_info_dict.items():
+            if group_dict[invoice.PREPAY_MANAGED_FIELD]:
+                managed_groups_list.append(group_name)
+
+        credits_mask = (
+            self.prepay_credits[invoice.PREPAY_MONTH_FIELD] == self.invoice_month
+        ) & (
+            self.prepay_credits[invoice.PREPAY_GROUP_NAME_FIELD].isin(
+                managed_groups_list
+            )
+        )
+        return self.prepay_credits[credits_mask]
+
+    def _export_prepay_credits_snapshot(self, credits_snapshot):
+        credits_snapshot.to_csv(
```

Comment:
This should also upload to s3.

Reply:
The Credits snapshot will now to uploaded to `f"Invoices/{self.invoice_month}/NERC_Prepaid_Group-Credits-{self.invoice_month}.csv"` and the archive location `f"Invoices/{self.invoice_month}/Archive/NERC_Prepaid_Group-Credits-{self.invoice_month} {util.get_iso8601_time()}.csv"`